examples/automatic_speech_recognition/automatic_speech_recognition/seed_test_run.py

```diff
@@ -0,0 +1,162 @@
+}
+
+MODEL_A = {
+    "model family": "whisper",
+    "model name": "whisper-1-default",
+    "description": "OpenAI's premier speech-to-text model.",
+}
+
+MODEL_B = {
+    "model family": "whisper",
+    "model name": "whisper-1-translate",
+    "description": "OpenAI's premier speech-to-text model w/ English translations.",
+}
+
+MODEL_C = {
+    "model family": "wav2vec2",
+    "model name": "wav2vec2-base-960h",
+    "description": "Facebook's Wav2Vec2 model trained on 960h of LibriSpeech audio samples.",
+}
+
+MODEL_METADATA: Dict[str, Dict[str, str]] = {
+    "whisper-1-default": MODEL_A,
+    "whisper-1-translate": MODEL_B,
+    "wav2vec2-base-960h": MODEL_C,
+}
+
+
+def seed_test_run(
+    mod: Tuple[str, str],
+    test_suite: TestSuite,
+    infs: pd.DataFrame,
+) -> None:
+    def get_stored_inferences(
+        df: pd.DataFrame,
+    ) -> Callable[[TestSample], Inference]:
+        def infer(sample: TestSample) -> Inference:
+            result = df.loc[df["id"] == sample.metadata['file_id']]
+            transcription = result[f"inference_{mod[0]}"].values[0]
+            return Inference(
+                transcription=ClassificationLabel(transcription)
+            )
+
+        return infer
+
+    model_name = mod[0]
+    print(f"working on {model_name} and {test_suite.name} v{test_suite.version}")
+    infer = get_stored_inferences(infs)
+    model = Model(f"{mod[1]}", infer=infer, metadata={**MODEL_METADATA[model_name], **COMMON_METADATA})
+    print(model)
+
+    test(model, test_suite, evaluate_audio_recognition, reset=True)
+
+
+def main(args: Namespace) -> None:
+    kolena.initialize(verbose=True)
+
+    mod = MODEL_MAP[args.model]
+    print("loading inference CSV")
+    s3_path = f"s3://{BUCKET}/{DATASET}/inference.csv"
+    csv_to_use = s3_path if args.local_csv is None else args.local_csv
+    columns_of_interest = [
+        "id",
+        "locator",
+        "text",
+        "inference_whisper-1-default",
+        "inference_whisper-1-translate",
+        "inference_wav2vec2-base-960h",
+    ]
+    df_results = pd.read_csv(csv_to_use, usecols=columns_of_interest)
+
+    if args.test_suite is None:
+        print("loading test suite")
+        test_suites = [TestSuite.load(name) for name in TEST_SUITE_NAMES]
+        for test_suite in test_suites:
+            print(test_suite.name)
+            seed_test_run(mod, test_suite, df_results)
+    else:
+        test_suite = TestSuite.load(args.test_suite)
+        print(test_suite.name)
+        seed_test_run(mod, test_suite, df_results)
+
+
+if __name__ == "__main__":
+    ap = ArgumentParser()
+    ap.add_argument(
+        "model",
+        type=str,
+        choices=sorted(MODEL_MAP.keys()),
+        help="The name of the model to test.")
+    ap.add_argument(
+        "--test-suite",
+        type=str,
+        help="Optionally specify a test suite to test. Test against all available test suites when unspecified.",
+    )
+    ap.add_argument(
```

Comment:
I don't think local csv option is necessary for an example.  Let's drop this argument